packages/esm-patient-conditions-app/src/conditions/conditions.resource.tsx

```diff
@@ -26,7 +26,7 @@ export type CodedCondition = {
 
 export function useConditions(patientUuid: string) {
   const { data, error, isValidating } = useSWR<{ data: FHIRConditionResponse }, Error>(
-    `${fhirBaseUrl}/Condition?patient=${patientUuid}`,
+    `${fhirBaseUrl}/Condition?patient=${patientUuid}&_count=500`,
```

Comment:
Hey @hadijahkyampeire , shouldn't we have pagination in place instead of calling count of 500.

By default, it should only fetch 10 conditions, and when user clicks on the next arrow, it should fetch another set of results. Also, in the fhir module, we get a `total` variable giving us a total results, and hence we can use pagination more effectively.


Consider this, if there are around 100s of conditions, they'll be fetched all together everytime, even if the user doesn't look at the conditions widget at all. This take up memory and processing, where it is not even needed.

Reply:
And bandwidth too. :)

Reply:
@vasharma05 @dkayiwa as you see I have a commit for removing pagination, so no we don't want pagination we just want to request for more results from the backend, please read the ticket requirements and the proposed solution.

Reply:
The allowance for pagination according to @gracepotma  should be maybe showing 100 results per page, I can maybe put back the pagination with 100 per page but I would say we just leave it for now because the table can scroll and we don't want to hide important conditions behind pagination.